lms/djangoapps/mobile_api/settings.py

```diff
@@ -0,0 +1,10 @@
+'''
+	Settings for Mobile Api
+'''
+
+FB_SETTINGS = {	'_FACEBOOK_APP_SECRET' : "6c26348ef355f53a531890e980ddc731",
```

Comment:
You can declare these settings in the `lms/envs/common.py` and `lms/envs/aws.py` files as mentioned in another comment.  The actual values of those settings will then be set in a JSON file managed by a private repo accessible to our devOps.